pkg/integration/tests/commit/amend.go

```diff
@@ -0,0 +1,29 @@
+package commit
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var AmendHead = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Amending head.",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.
+			CreateNCommits(5). // these will appear as commit 05, 04, 04, down to 01
```

Comment:
Well the docs are actually there, I just copied this part from your tests because why not, so here we are. The comment doesn't really help with anything, I just didn't remove it.